CONTRIBUTING.md

```diff
@@ -0,0 +1,98 @@
+# Contributing
+First off, thank you for considering contributing to Learn Anything. It's people
+like you that make Learn Anything such a great platform (and community).
+
+The following guidelines help communication between maintainers of the project,
+avoid wasted effort, and should make it as easy as possible for you to make the
+changes that you want to see.
+
+There are many ways you can contribute to this project: improving documentation
+(including these guidelines), submitting bug reports, or writing code that will
+be incorporated in the site.
+
+
+## Contribution Workflow
+### 1. Search Issue
+If you want to contribute to the project, first of all, search if there is an
+issue describing the problem that you encountered or the changes that you want
+to see.
+
+**If there is an issue and:**
+- you have additional information to add to it => please comment
+- you want to work on it and there is no one assigned => go to [step 3.](### 3. Assign Issue)
+
+
+### 2. Create Issue
+If you didn't find any relevant issue, create one yourself and use appropriate
+labels.
+
+Currently we use the following labels:
+- documentation
+- feature
+- bug
+
+**Documentation**
+1. Describe what is unclear or missing.
+
+**Feature**
+1. What feature do you want to see?
+2. Why do you want to see this feature?
+3. How will it benefit other users?
+
+**Bug**
+1. What actions did you take?
+2. What was the expected outcome?
+3. What happened instead?
+4. Add any additional details that could be useful.
+
+
+### 3. Assign Issue
+You decided to work on an issue:
+- If there is already someone assigned, please don't work on it. Rather comment
+and ask how you can help.
+- If no one is assigned to it, assign yourself.
```

Comment:
I believe only maintainers can assign people to issues. Maybe should be mentioned?

Reply:
Okay didn't know, I'll change this part then. What do you think would be a good way to do this? People just comment saying that they're going to work on issues and we assign them?

Reply:
Yes. I think that would be best.